_data/tutorials.yaml

```
@@ -781,4 +782,36 @@ creating-first-apache-kafka-streams-application:
   introduction: "You'd like to get started with Kafka Streams, but you're not sure where to start. In this tutorial you'll build a small stream processing application and produce some sample data to test it. You can use the code in this tutorial as an example of how to use Kafka Streams. After you complete this tutorial, you can go more in-depth in the <a href='https://developer.confluent.io/learn-kafka/kafka-streams/get-started/'>Kafka Streams 101 Course</a>."
   status:
     confluent: enabled
-    kstreams: enabled
\ No newline at end of file
+    kstreams: enabled
+
+multiple-event-types:
+  title: "Handling multiple event types in a topic"
+  canonical: confluent
+  meta-description: "handling multiple event types"
+  slug: "/multiple-event-type-topics"
+  question: "How can I have multiple event types in a topic and maintain topic-name subject constraints?"
```

Switching to `you` https://github.com/confluentinc/kafka-tutorials/pull/982/files

```suggestion
  question: "How can you have multiple event types in a topic and maintain topic-name subject constraints?"
```